packages/gatsby-plugin-netlify-cms/src/gatsby-node.js

```diff
@@ -173,7 +173,7 @@ exports.onCreateWebpackConfig = (
             messages: [
               `Netlify CMS is running at ${
                 program.https ? `https://` : `http://`
-              }${program.host}:${program.proxyPort}/${publicPathClean}/`,
+              }${program.host}:${program.port}/${publicPathClean}/`,
```

Comment:
I think we should not change it in plugin yet given that core will keep `proxyPort` on `program` for now and instead have TODO for v5 (similar as one in core) - just so it doesn't start showing incorrect info for older v4 versions